security-release-process.md

```diff
@@ -0,0 +1,65 @@
+
+### Fix Team Organization
+These steps should be completed within the first 24 hours of disclosure.
+
+The Fix Lead will work quickly to identify relevant engineers from the affected projects and packages and CC those engineers into the disclosure thread. These selected developers are the Fix Team.
+The Fix Lead will get the Fix Team access to private security repos to develop the fix.
+
+### Fix Development Process
+These steps should be completed within the 1-7 days of Disclosure.
+
+The Fix Lead and the Fix Team will create a CVSS using the CVSS Calculator. The Fix Lead makes the final call on the calculated CVSS; it is better to move quickly than make the CVSS perfect.
+The Fix Team will notify the Fix Lead that work on the fix branch is complete once there are LGTMs on all commits in the private repo from one or more maintainers.
+If the CVSS score is under 4.0 (a low severity score) the Fix Team can decide to slow the release process down in the face of holidays, developer bandwidth, etc. These decisions must be discussed on the gardener-security@googlegroups.com mailing list.
+
+### Fix Disclosure Process
+With the fix development underway, the Fix Lead needs to come up with an overall communication plan for the wider community. This Disclosure process should begin after the Fix Team has developed a Fix or mitigation so that a realistic timeline can be communicated to users.
+
+Disclosure of Forthcoming Fix to Users (Completed within 1-7 days of Disclosure)
+The Fix Lead will email gardener@googlegroups.com informing users that a security vulnerability has been disclosed and that a fix will be made available at YYYY-MM-DD HH:MM UTC in the future via this list. This time is the Release Date.
+The Fix Lead will include any mitigating steps users can take until a fix is available.
+The communication to users should be actionable. They should know when to block time to apply patches, understand exact mitigation steps, etc.
+
+### Fix Release Day (Completed within 1-21 days of Disclosure)
+The Release Managers will ensure all the binaries are built, publicly available, and functional before the Release Date. 
+The Release Managers will create a new patch release branch from the latest patch release tag + the fix from the security branch. As a practical example if v1.5.3 is the latest patch release in gardener.git a new branch will be created called v1.5.4 which includes only patches required to fix the issue.

```

Comment:
> The Release Managers will create a new patch release branch from the latest patch release tag + the fix from the security branch.

I don't understand this sentence

Above you mention that a private repository is used to develop security patches. Can you make clear which steps happen in these private repositories ?

Reply:
To avoid too early disclosure of a vulnerability to the public the complete security fix shall be developed in a private repository